src/main/java/Duke.java

```diff
@@ -19,16 +19,16 @@ public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
         ArrayList<Task> tasks = new ArrayList<>();
 
-        while(true){
+        while (true) {
```

Comment:
If there's a condition to exit the program, would it be better to place the condition inside here? In terms of readability. 
while(! isByeCommand). 
Not saying is wrong, but it would beneficial to people reading the code.